### PR TITLE
Doc and readme updates

### LINKED ISSUE #189: Update Readme.md
Update content in https://github.com/include-dcc/include-linkml/blob/v3.0.0_rc1/README.md to reflect changes in v3

### PR DESCRIPTION
**Pull request by madanucd**
Closes #189 and #190

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,65 @@
...
 
-This repository includes the LinkML model and tooling for the v2 Data Model
+# INCLUDE LinkML Data Model
 
-# Where to find the data model:
+## Overview
 
-* The YAML files are in `src > linkml`
-* The JSON-LD schema is in `src > data > schematic`
+This repository hosts the Include LinkML Model, a specialized data model crafted to empower collaboration and streamline data integration within Down syndrome research. By harnessing advanced [LinkML](https://linkml.io/) features, it provides a versatile framework for harmonizing data representation, articulating metadata, and delineating intricate relationships between entities. This structured approach not only promotes seamless data exchange but also facilitates interoperability and scalability across diverse research initiatives.
```
Comment (twhetzel) [inline README.md:7]: nit: Include --> INCLUDE

Comment (madanucd) [reply README.md]: Thanks @twhetzel ! Updated "Include" to "INCLUDE" throughout the docs.

File: src/docs/about.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+
+### How to Get Involved
+- **Explore the Model:** Gain a deeper understanding of essential metadata elements for Down syndrome research.
+  
+- **Collaborate:** Join forces to leverage the INCLUDE LinkML Model and accelerate progress.
+  
+- **Contribute:** Share expertise, insights, and data to enrich the model and shape the future of Down syndrome research.
+
+### Start Exploring
+Visit [INCLUDE Data Coordinating Center (DCC)](https://includedcc.org/) to access the Include LinkML Model and join the collaborative effort toward transforming Down syndrome research and healthcare.
```
Comment (twhetzel) [inline src/docs/about.md:41]: @madanucd thanks! Looks good other than this one "...the Include LinkML Model..."

**Review by twhetzel — DISMISSED** [review]
Looks good other than one update. See comment inline. 

Very nice documentation!!!

**Comment by lopierra** [comment]
@madanucd thank you for doing this - it looks great! My only question is whether we can delete the [last section of the readme](https://github.com/include-dcc/include-linkml/tree/doc_update?tab=readme-ov-file#setting-up-include-schemasheets-with-schematic) that mentions schematic?

**Review by lopierra — APPROVED** [review]

**Comment by madanucd** [comment]
@twhetzel and @lopierra, thank you both for your valuable feedbacks!

**Pull request merged**